wandb_init.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python
+# -*- coding: utf8 -*-
+"""
+
+@date: 07.02.23
+@author: leonhard.hennig@dfki.de
+"""
+
+import os
+from importlib.util import find_spec
+
+
+def prepare_wandb():
+    # if wandb is installed
+    if find_spec("wandb"):
+        import wandb
+        wandb_api_key = os.getenv("WANDB_API_KEY", None)
+        if wandb_api_key is not None:
+            print('Logging into wandb')
+            wandb.login(key=wandb_api_key)
```

Comment:
Is this method called in any way? Maybe we do just this:
```suggestion
# if wandb is installed
if find_spec("wandb"):
    import wandb
    wandb_api_key = os.getenv("WANDB_API_KEY", None)
    if wandb_api_key is not None:
        print('Logging into wandb')
        wandb.login(key=wandb_api_key)
```

Reply:
i just copied the code you sent me in Teams - I'm fine with changing it ;-)

Reply:
👍 Ok, can you test if it also works for you with this proposed change?

Reply:
yes, worked in multiple runs

Reply:
thanks :)